view/frontend/templates/js/replacejs.phtml

```diff
@@ -1270,6 +1270,13 @@ if (!$block->isSaveCartInSections()) { ?>
         'mage/cookies',
         'domReady!'
     ], function ($, authenticationPopup, customerData) {
+        var sectionDataIds = $.cookieStorage.get('section_data_ids');
+        if (sectionDataIds && sectionDataIds.b  !== undefined) {
+            $.cookieStorage.set('section_data_ids','');
+            storageInvalidation = $.initNamespaceStorage('mage-cache-storage-section-invalidation').localStorage;
+            storageInvalidation.removeAll();
+        }
+
```

Comment:
@vitaliyreznikov I was curious what happens when you don't include this part. Is it really needed after the store is updated and caches cleared? Full page cache included. Did not get a chance to test. Did you?

Reply:
Yes, I tested very carefully. Yes, we need it if caches cleared on server side, but user has bad cookies.
Few users can be affected, only who made purchase and continue actively using site (cookies are deleted after 1 day of inactivity), but I offered to include this check just in case.